src/zui/ZUIPersonSelect.tsx

```diff
@@ -215,12 +237,19 @@ const MUIOnlyPersonSelect: FunctionComponent<ZUIPersonSelectProps> = (
               }}
             >
               {children}
-              {!disabled && (
+              {!disabled && showAddPersonButton && (
```

Comment:
To follow up on my previous comment, what if this used `wasLoading` instead (and `wasLoading` was not set to false on line 215).

```suggestion
              {!disabled && wasLoading && (
```

Reply:
✅